netlify.toml

```diff
@@ -33,6 +33,16 @@
   to = "/oracles/oracle-data"
   status = 301
   force = true
+[[redirects]]
+  from = "/solana//migration/hotspot-owner"
+  to = "/oracles//migration/hotspot-operator"
+  status = 301
+  force = true
+[[redirects]]
+  from = "/solana//migration/maker"
+  to = "/oracles//migration/hotspot-maker"
```

Comment:
is a `//` needed here in these changes?

Reply:
ooh these should not have the double "//" good catch, that would have been annoying